kafkacat/README.adoc

```diff
@@ -11,11 +11,11 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
 :community:
 :imagesdir: ./images
-:product-long: bf2fc6cc711aee1a0c2a
-:product: bf2
+:product-long: app-services
```

Comment:
```suggestion
:product-long: Application Services
```
![Screenshot 2021-04-09 at 14 25 39](https://user-images.githubusercontent.com/981838/114187076-82b7b900-993f-11eb-8913-64df4bad6e84.png)


![Screenshot 2021-04-09 at 14 25 39](https://user-images.githubusercontent.com/981838/114187066-7fbcc880-993f-11eb-912d-b6db28a8932b.png)

Reply:
Application services and short ones can be App Services. 
![Screenshot 2021-04-09 at 14 27 24](https://user-images.githubusercontent.com/981838/114187317-bf83b000-993f-11eb-9a2e-dee2e3f287ab.png)